pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_identifying-module-types.adoc

```diff
@@ -1,47 +1,54 @@
 [id='identifying-module-types_{context}']
-= Identifying {ProductShortName} module types
-:pantheon-module-type: REFERENCE
+= Adding the module type to modules in {ProductShortName}
+:_module-type: REFERENCE
 
 [role="_abstract"]
-In {ProductShortName}, each module must be identified by type: concept, procedure, or reference.
+In {ProductShortName}, you can search and filter modules  by type: concept, procedure, or reference. You declare a module's type by adding an identifier to each module.
 
-For information about module types, see https://redhat-documentation.github.io/modular-docs/#writing-mod-docs[Types of modules in Red Hat documentation].
+Use one or both of the following identifiers to declare the module's type:
 
-Identify the module type by using either one or both of the following:
-
-* The correct attribute after the module's title
-* The correct prefix in the module's name
-
-Using the correct attribute (preferred method)::
+* The `:_module-type:` attribute after the module's title
+* The corresponding prefix in the module's file name
 
+Attribute (preferred method)::
 Enter the correct attribute in the line immediately following the module's title, choosing among the following:
-
-* `:pantheon-module-type: CONCEPT`
-* `:pantheon-module-type: PROCEDURE`
-* `:pantheon-module-type: REFERENCE`
-
++
+* `:_module-type: CONCEPT`
+* `:_module-type: PROCEDURE`
+* `:_module-type: REFERENCE`
++
 ====
 .Example
 
-`= Identifying {ProductShortName} module types :pantheon-module-type: REFERENCE`
+`= Identifying {ProductShortName} module types
```

Comment:
Both the lines are rendered on the same line.

I suggest:

`= Identifying {ProductShortName} module types`
`:_module-type: REFERENCE`

Reply:
Done.